target/linux/ath79/dts/qca9563_zte_mf286.dts

```diff
@@ -66,13 +66,20 @@
 			#address-cells = <1>;
 			#size-cells = <1>;
 
+			/* ath9k */
 			cal_caldata_1000: cal@1000 {
 				reg = <0x1000 0x440>;
 			};
 
+			/* ath10k wave1 in plain MF286 */
 			cal_caldata_5000: cal@5000 {
 				reg = <0x5000 0x844>;
 			};
+
+			/* ath10k wave2 in early MF286A */
+			precal_caldata_5000: precal@5000 {
+				reg = <0x5000 0x2f20>;
+			};
```

Comment:
These blocks are handling the same reg range. Does the kernel accept this?

Reply:
The same as in the PR for `main` - yes, kernel is fine with that.